src/main/resources/application.properties

```diff
@@ -18,7 +18,12 @@ spring.jpa.open-in-view=false
 # Kafka
 spring.kafka.bootstrap-servers=${BOOTSTRAP_SERVER}
 spring.kafka.template.default-topic=${TIMECARD_TOPIC:callisto-timecard-timeentries}
-spring.kafka.security.protocol=SSL
-spring.kafka.ssl.key-store-location=file://timecard-restapi-keystore/timecard-restapi.keystore.jks
+spring.kafka.properties.security.protocol=SSL
+spring.kafka.ssl.key-store-location=${KEYSTORE_LOCATION:file://timecard-restapi-keystore/timecard-restapi.keystore.jks}
+spring.kafka.ssl.trust-store-location=${TRUSTSTORE_LOCATION}
```

Comment:
Have we checked having a null env variable for truststore will work in a branch deploy?